crates/astria-proto/proto/astria/sequencer/v1/transaction.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package astria.sequencer.v1;
+
+import "astria/primitive/v1/types.proto";
+
+message SignedTransaction {
+    bytes signature = 1;
+    bytes public_key = 2;
+    UnsignedTransaction transaction = 3;
+}
+
+message UnsignedTransaction {
+    oneof value {
+        AccountsTransaction accounts_transaction = 1;
+    }   
+}
+
+message AccountsTransaction {
```

Comment:
Judging from the implementation code, it is an error to leave e.g. `amount` unset. Can you also add that information here?

Reply:
you mean having a zero-value transfer (or leaving the other values as zero)? that's allowed, except for nonce, which should be 1 for an account's first tx

Reply:
I meant from a protobuf3 (+ rust) point of view. Since in protobuf3 technically all fields are optional (meaning it's not an error to not send them), Rust represents `amount` as an `Option`.

I think creating an `AccountsTransaction` without `amount` set is invalid from a business logic point of view, so I think we should document this here (probably in all our protobufs, but here is the first time I really noticed).